packages/Charts/src/demos/demo.ts

```diff
@@ -0,0 +1,71 @@
+import * as DG from 'datagrok-api/dg';
+import * as grok from 'datagrok-api/grok';
+
+
+export class Demo {
+  static async chordViewerDemo() {
+    const df = await grok.data.getDemoTable('energy_uk.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('ChordViewer');
+  }
+
+  static async globeViewerDemo() {
+    const df = await grok.data.getDemoTable('geo/earthquakes.csv');
+    const tableView = grok.shell.addTableView(df);
+    // semantic type detection is needed for the latitude, longitude and magnitude columns
+    DG.debounce(df.onSemanticTypeDetected, 50).subscribe((_) => tableView.addViewer('GlobeViewer'));
+  }
+
+  static async groupAnalysisViewerDemo() {
+    const df = await grok.data.getDemoTable('demog.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('GroupAnalysisViewer');
+  }
+
+  static async radarViewerDemo() {
+    const df = await grok.data.getDemoTable('demog.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('RadarViewer');
+  }
+
+  static async sankeyViewerDemo() {
+    const df = await grok.data.getDemoTable('energy_uk.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('SankeyViewer');
+  }
+
+  static async sunburstViewerDemo() {
+    const df = await grok.data.getDemoTable('demog.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('SunburstViewer');
+  }
+
+  static async surfacePlotDemo() {
+    const df = await grok.data.getDemoTable('demog.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('SurfacePlot');
+  }
+
+  static timelinesViewerDemo() {
+    const df = DG.DataFrame.fromCsv(
+      `USUBJID, AESTDY, AEENDY, SEX, AGE
+      s1, 10/02/2018, 10/09/2018, F, 48
+      s2, 10/04/2018, 10/07/2018, M, 51
+      s3, 10/02/2018, 10/05/2018, F, 39
+      s4, 10/07/2018, 10/08/2018, M, 43`);
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('TimelinesViewer');
```

Comment:
@onuf , please come up with a better example that would showcase most of the timelines functionality

Reply:
done